pex/bin/pex.py

```diff
@@ -551,33 +478,34 @@ def _safe_link(src, dst):
   os.symlink(src, dst)
 
 
-def build_pex(args, options, resolver_option_builder):
-  with TRACER.timed('Resolving interpreters', V=2):
-    def to_python_interpreter(full_path_or_basename):
-      if os.path.exists(full_path_or_basename):
-        return PythonInterpreter.from_binary(full_path_or_basename)
+def build_pex(reqs, options):
+  interpreters = None  # Default to the current interpreter.
+
+  # NB: options.python and interpreter constraints cannot be used together.
+  if options.python:
+    with TRACER.timed('Resolving interpreters', V=2):
+      def to_python_interpreter(full_path_or_basename):
+        if os.path.exists(full_path_or_basename):
+          return PythonInterpreter.from_binary(full_path_or_basename)
+        else:
```

Comment:
Certainly, if you view the result as easier reading. This seems subjective and we don't have a style guide I know of providing direction here beyond local style. Trumping to me is the fact that this keeps the prior style, it just indents the block and github diff is fairly rudimentary failing to display this fact.

Reply:
To me, it's much more readable because this is highly indented code. But I agree that we don't have a consistent style guide and of course won't block for something like this. Nor is this probably worth a new CI run unless you include the change among other changes.